configs/custom_weight_gh_config.yaml

```diff
@@ -13,23 +13,23 @@ graphhopper:
     - name: car
       vehicle: car
       weighting: custom
-      custom_model_file: empty
+      custom_model_files: []
```

Comment:
This is the new way of referencing custom model files in configs, I've updated everywhere accordingly